package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "wthr",
```

The name should be the same across the entire project.
```suggestion
  "name": "wather-app",
```

---

+

---

?